<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkInteractiveTransformationWidgetControls</class>
 <widget class="QWidget" name="QmitkInteractiveTransformationWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="interactiveTranslationBox">
     <property name="title">
      <string>Interactive Translation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="xTranslationLabel_2">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>x-Direction
(Frontal):       </string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_XTransSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="minimum">
             <number>-500</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_XTransSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="yTranslationLabel_2">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>y-Direction
(Sagittal):      </string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_YTransSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>-500</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_YTransSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="zTranslationLabel_2">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>z-Direction
(Transversal):</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_ZTransSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>-500</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="sliderPosition">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_ZTransSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-500.000000000000000</double>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="interactiveRotationBox">
     <property name="title">
      <string>Interactive Rotation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="xRotationLabel">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>x-Axis
(Frontal):       </string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_XRotSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>45</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_XRotSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="yRotationLabel">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>y-Axis
(Sagittal):      </string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_YRotSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>45</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_YRotSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="zRotationLabel">
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>z-Axis
(Transversal):</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="m_ZRotSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksAbove</enum>
            </property>
            <property name="tickInterval">
             <number>45</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="m_ZRotSpinBox">
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="m_UseManipulatedToolTipPB">
          <property name="text">
           <string>Use Manipulated ToolTip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_ResetPB">
          <property name="text">
           <string>Reset To Identity</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_RevertChanges">
          <property name="text">
           <string>Revert Changes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_Cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
